### PR TITLE
Fix Docker logs for different Rails envs

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -10,7 +10,9 @@ DQT_CLIENT_PORT=13
 ADMIN_ALLOWED_IPS=::1,127.0.0.1
 
 GECKOBOARD_API_KEY=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
-ENVIRONMENT_NAME=test
+ENVIRONMENT_NAME=development
+
+RAILS_ENV=development
```
Comment (WorkSutton) [inline .env.test:15]: @nomasprime @and-lucas-kelly Should these not be test?

**Comment by WorkSutton** [comment]
@nomasprime @and-lucas-kelly Do we still need this PR? Pretty nervous about all the changes to the azure-pipelines file, as believe these were simply to replicate the issue that was being investigated with DQT?

File: .env.test (hunk 1 above)
Comment (and-lucas-kelly) [reply .env.test:15]: it should be test

File: Dockerfile [hunk 2]
```diff
@@ -168,11 +170,11 @@ RUN chown -hR appuser:appgroup ${APP_HOME}/log
 RUN chown -hR appuser:appgroup ${APP_HOME}/app
 RUN chown -hR appuser:appgroup ${APP_HOME}/tmp
 
-RUN touch ${APP_HOME}/log/production.log
+RUN touch ${APP_HOME}/log/${RAILS_ENV}.log
```
Comment (and-lucas-kelly) [inline Dockerfile:174]: happy for these changes to get merged or i can update in a separate branch

File: azure-pipelines.yml [hunk 3]
```diff
@@ -18,16 +18,16 @@ steps:
...
 
-      docker-compose --file=docker-compose.test.yml build
-    displayName: Build test Docker image
+  #     docker-compose --file=docker-compose.test.yml build
+  #   displayName: Build test Docker image
 
-  - script: docker-compose --file=docker-compose.test.yml run --rm test
-    displayName: Run tests on Docker
+  # - script: docker-compose --file=docker-compose.test.yml run --rm test
+  #   displayName: Run tests on Docker
```
Comment (and-lucas-kelly) [inline azure-pipelines.yml:31]: These changes definitely should not get merged

File: docker-compose.test.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@ services:
         - dfedigital/teacher-payments-service:cache-test-dependencies
       target: dependencies
       args:
-        RAILS_ENV: test
+        RAILS_ENV: development
     image: local/dfe-teachers-payment-service:test-dependencies
```
Comment (and-lucas-kelly) [inline docker-compose.test.yml:12]: These should be reverted back to test as well

**Review by and-lucas-kelly — APPROVED** [review]
reverted three files that needed to be rolled back

**Pull request merged**